sdk/pinocchio/src/instruction.rs

```diff
@@ -87,7 +94,8 @@ impl<'a> From<&'a AccountInfo> for Account<'a> {
             data_len: account.data_len() as u64,
             data: offset(account.raw, 88),
             owner: offset(account.raw, 40),
-            rent_epoch: 0,
+            rent_epoch: 0, // This is probably ok, but the rent epoch is in the
+                           // account info and could be used!
```

Comment:
We are not reading the `epoch` value since we can't get it zero-copy – it appears after the variable data of the account.